docs/openapi/components/schemas/credentials/InvoiceCredential.yml

```diff
@@ -1,7 +1,7 @@
 $linkedData:
-  term: CommercialInvoiceCredential
-  '@id': https://w3id.org/traceability#CommercialInvoiceCredential
-title: Commercial Invoice Credential
+  term: InvoiceCredential
```

Comment:
@BenjaminMoe, I appreciate that you want to avoid definition redundancy - similar to how they used to share `$ref:Invoice.yml`. But from a "recognizable by the industry" point of view, this seems like a step backwards. People are used to forms with a "Commercial Invoice" header or "Proforma Invoice" header. I suggest we accept redundancy and keep the individual types of credentals.

Reply:
PR moved to https://github.com/w3c-ccg/traceability-vocab/pull/808